src/loginForm.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<html>
+
+<head>
+    <style>
+        form {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            display: block;
+            width: 100%;
+            max-width: 400px;
+            background-color: #FFF;
+            margin: 0;
+            padding: 2.25em;
+            box-sizing: border-box;
+            border: solid 1px #DDD;
+            border-radius: .5em;
+            font-family: 'Source Sans Pro', sans-serif;
+        }
+
+        .inputGroup {
+            margin: 0 0 2em;
+            padding: 0;
+            position: relative;
+        }
+
+        label {
+            margin: 0 0 12px;
+            display: block;
+            font-size: 1.25em;
+            color: #217093;
+            font-weight: 700;
+            font-family: inherit;
+        }
+
+        input {
+            font-size: 1.5em;
+            width: 100%;
+        }
+
+        #loginBtn {
+            width: 100%;
+            height: 2em;
+            font-size: 1.5em;
+            font-weight: 600;
+            background: dodgerblue;
+            border-radius: .5em;
+            border: none;
+            color: white;
+        }
+
+        #loginBtn:active {
+            background: deepskyblue;
+        }
+    </style>
+</head>
+
+<body>
+    <form>
+        <div class="inputGroup inputGroup1">
+            <label for="email">E-mail</label>
+            <input type="text" id="email" placeholder="Input E-mail address" />
+        </div>
+        <div class="inputGroup inputGroup2">
+            <label for="password">Password</label>
+            <input type="password" id="password" placeholder="Input password" />
+        </div>
+        <div>
+            <button id="loginBtn" type="button">Log In</button>
+        </div>
+    </form>
+</body>
+
+</html>
+<script>
+
+    const addressInput = document.getElementById('email');
+    const passwordInput = document.getElementById('password');
+    const loginBtn = document.getElementById('loginBtn');
+
+    const login = () => {
+        if (!!addressInput.value) {
```

Comment:
```suggestion
        if (addressInput.value) {
```
와 같은 코드입니다. if문은 boolean 타입이 아니어도 truthy하면 boolean타입의 True값처럼 받아들여요.

Reply:
이 아래쪽에 있는 !! 도 똑같다고 생각해주시면 됩니다 'ㅅ'

Reply:
저도모르게 불편하게 작성하고 있었네요~ 😂